.claude/commands/gh-review.md

```diff
@@ -0,0 +1,9 @@
+Address feedback and checks in a Pull Request.
```

Comment:
this command effectively is like "review the feedback and help me work through it" - works pretty well, but may need tweaked to be more aware of what checks we run and what they mean